classes/task/sendfailednotifications_task.php

```diff
@@ -183,32 +184,32 @@ public function execute() {
         $manager = get_log_manager();
         $store = new store($manager);
 
-        echo "In send failed notifications task execute".PHP_EOL;
+        echo "In send failed notifications task execute" . PHP_EOL;
```

Comment:
Could you add this to the string file?

Reply:
Do we even need this line @mlynn-lp?

Reply:
No it's just to show some output when running on the cli.